CHANGELOG.md

```diff
@@ -3,6 +3,9 @@ Next Release
 
 #### Features
 
+* [#1062](https://github.com/intridea/grape/issues/1062): Added headers
```

Comment:
Put this on 1 line and match the syntax from below (no period before `-`, quote `ValidationErrors`).

Maybe we can phrase this better, "Fix: validation errors will include headers set by `header`"?